doc/stdenv/stdenv.chapter.md

```diff
@@ -1411,6 +1411,7 @@ these in the [Hooks Reference](#chap-hooks).
 ### Compiler and Linker wrapper hooks {#compiler-linker-wrapper-hooks}
 
 If the file `${cc}/nix-support/cc-wrapper-hook` exists, it will be run at the end of the [compiler wrapper](#cc-wrapper).
+If the file `${binutils}/nix-support/ld-wrapper-hook` exists, it will be run at the end of the linker wrapper, before the linker runs.
 If the file `${binutils}/nix-support/post-link-hook` exists, it will be run at the end of the linker wrapper.
```

Comment:
Within nixpkgs, this appears to be used by `pkgs/os-specific/darwin/signing-utils/post-link-sign-hook.nix` only.

https://github.com/NixOS/nixpkgs/blob/4168283a5b23ead5fe0e823195a42e66f1f25392/pkgs/os-specific/darwin/signing-utils/post-link-sign-hook.nix#L1-L17

Reply:
This hook is nearly vestigial. Darwin only uses this hook early in the bootstrap. Once it builds a version of ld64 supporting code-sigining, it’s no longer used. It can be dropped once Darwin’s bootstrap tools are updated with that version of ld64, which should hopefully be doable after this staging-next merge. Getting clang 19 into the bootstrap is also important for other reasons (such as guaranteeing C23 support in the bootstrap).